src/main/scala/com/adobe/api/platform/runtime/metrics/PrometheusRecorder.scala

```diff
@@ -90,26 +81,13 @@ object PrometheusRecorder extends MetricRecorder with PrometheusExporter with Pr
 
   private def seconds(timeInMillis: Long) = TimeUnit.MILLISECONDS.toSeconds(timeInMillis)
 
-  private def counter(name: String, help: String, tags: String*) =
-    Counter
-      .build()
-      .name(name)
-      .help(help)
-      .labelNames(tags: _*)
-      .register()
-
-  private def histogram(name: String, help: String, tags: String*) =
-    Histogram
-      .build()
-      .name(name)
-      .help(help)
-      .labelNames(tags: _*)
-      .register()
+  private def createSource() =
+    Source.combine(createJavaClientSource(), createKamonSource())(Concat(_)).map(ByteString(_))
```

Comment:
I think it's a good idea to have support for both types of metrics, but I'm not sure when the Kamon API is going to be used in this `user-events` specific application.

Reply:
I forsee few usecases

1. Metric related to `user0events` service itself. Like rate of activation event records processed, kafka consumer metrics
2. Non action specific metric like those to track kind and memory stats. Proportion of activations which are composition, cold starts <sup>1</sup>

<sup>1</sup> - though that can be extracted from summing up action cold start metrics but may be better to explicitly track that at global level